### PR TITLE
Updates tests to support WordPress 5.2.3

### PR DESCRIPTION
**Pull request by bgrgicak**
This PR updates all tests to supprot the latest version of WordPress (5.2.3).

### CODE REVIEW

**Comment by gedex** [comment]
Finally, 47 passing:

```
(update/tests-support-WP-5.2.3) → npm test

> wp-e2e-page-objects@0.9.0 test /Users/akeda/code/wp-e2e-page-objects
> cross-env NODE_CONFIG_DIR='./test/config' BABEL_ENV=commonjs mocha --require babel-core/register --recursive



  UserFlow
    ✓ allows to open any /wp-admin/ page and returns its page object (567ms)
    ✓ allows to create post (20587ms)
    ✓ allows to log the user out from /wp-admin/ (324ms)

  FrontPage
    ✓ display the page with expected title
    ✓ display sidebar when page is loaded
    widgets displayed in the sidebar
      ✓ has recent posts widget
      ✓ has recent comments widget

  PageMap
    ✓ should return full page URL with getPageUrl

  Page
    ✓ getCurrentUrl returns current URL
    ✓ getTitle returns current page title
    ✓ titleContains checks if passed pattern matches current page title
    ✓ hasText checks if passed text appears in current page (2078ms)
    ✓ visit will open new URL based on instance.url (6397ms)

  SinglePage
    ✓ allows you to comment as logged-in user (3945ms)

  WPAdminCategories
    ✓ can edit category (1750ms)
    ✓ can view category (1106ms)
    ✓ can delete category (648ms)

  WPAdminCategoryEdit
    ✓ can edit category (4146ms)

  WPAdminPluginInstall
    ✓ can search plugin "woocommerce" (2746ms)
    ✓ can install plugin "woocommerce" (68901ms)
    ✓ can activate new installed plugin "woocommerce" (6913ms)

  WPAdminPlugins
    ✓ has list of plugins
    ✓ can activate (3130ms)
    ✓ can deactivate (3117ms)
    ✓ redirect to plugin install page when clicking Add New (1804ms)

  WPAdminPostNew
    ✓ can create a new post (13203ms)

  WPAdminPosts
    ✓ can edit posts (8004ms)
    ✓ can view posts (3243ms)
    ✓ can trash posts (3277ms)

  WPAdminTagEdit
    ✓ can edit tag (5444ms)

  WPAdminTags
    ✓ can edit tag (3622ms)
    ✓ can view tag (2091ms)
    ✓ can delete tag (964ms)

  WPAdmin
    ✓ has admin menus (173ms)
    ✓ may opens another page when menu is clicked (5743ms)
    ✓ has some admin menus with submenus (1462ms)
    ✓ may opens another page when submenu is clicked (34197ms)
    ✓ has quick links in the admin bar (83ms)
    ✓ may opens another page when quick link is clicked (5797ms)
    ✓ has some quick links with submenus (581ms)
    ✓ may opens another page when quick link submenu is clicked (7020ms)
    ✓ has my account in the admin bar (91ms)
    ✓ opens edit profile page from "My Account > Edit My Profile" (745ms)
    ✓ may trigger admin notice after performing some actions (1124ms)
    ✓ log the user out when clicking "My Account > Log Out" (939ms)

  WPLogin
    ✓ allows user to log in (2098ms)
    redirects to the dashboard page after logged in
      ✓ has welcome panel in the dashboard


  47 passing (5m)
```

**Comment by gedex** [comment]
I added small changes to fix single run test command and the order of test in `WPAdmin`.

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 language: node_js
 node_js:
   - "10"
+addons:
+	apt:
+		packages:
+			- google-chrome-stable
```
Comment (gedex) [inline .travis.yml]: not a biggie; please use spaces like the rest of the lines

**Comment by bgrgicak** [comment]
I just noticed that woocommerce wasn't deleted after the plugin-install test so fixed also this.
Will merge now and move to wc-e2e-page-objects.

**Pull request merged**